README.md

```diff
@@ -1,16 +1,22 @@
 # tiltak-docker-compose
-Oppset for å kjøre docker compose i Team Tiltak
+Oppset for å kjøre docker compose i Team Tiltak med kafka. 
+
+
+
 
 ## Start alt
 ```
-docker compose up
+docker-compose up --remove-orphans
```

Comment:
`docker compose` uten `-` er hva de kule folka bruker. 😉 
Det vil si `docker compose` er den nye måten å skrive `docker-compose` på som kom i V2.

https://docs.docker.com/compose/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2